src/JWT/JWT.cs

```diff
@@ -264,7 +264,12 @@ public static void Verify(string payloadJson, string decodedCrypto, string decod
             var secondsSinceEpoch = Math.Round((DateTime.UtcNow - UnixEpoch).TotalSeconds);
             if (secondsSinceEpoch >= expInt)
             {
-                throw new TokenExpiredException("Token has expired.");
+               var tokenExpiredException = new TokenExpiredException("Token has expired.")
```

Comment:
Yeah, I feel you pain, I'm a tabs guy too! Don't get me started ;) Usually what I do is format then find and replace \t with "    ".